src/cirrus/builtins/functions/update-state/lambda_function.py

```diff
@@ -154,9 +152,10 @@ def parse_event(event):
     #   - error object
     try:
         if 'error' in event:
-            logger.debug('looks like a stac record with an error message, i.e., workflow-failed')
+            logger.debug('looks like a payload with an error message, i.e., workflow-failed')
             return (
                 ProcessPayload.from_event(event),
+                None,
```

Comment:
I recommend to documenting/ updating the return types

Reply:
Agreed, but again I plan future refactoring which will make this moot.